<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientGui</class>
 <widget class="QWidget" name="ClientGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car ctrl client</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Camera: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cameraDev">
          <property name="text">
           <string>/dev/video0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="setDeviceName">
        <property name="text">
         <string>Set device name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="isOpened">
        <property name="text">
         <string>Is opened</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="initDevice">
        <property name="text">
         <string>Init camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imageSize">
        <property name="text">
         <string>Image size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startCapture">
        <property name="text">
         <string>Start capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopCapture">
        <property name="text">
         <string>Stop capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acquire">
        <property name="text">
         <string>Acquire frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="data">
        <property name="text">
         <string>Read frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="release">
        <property name="text">
         <string>Release frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readFrame">
        <property name="text">
         <string>Get frame</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="lastError">
        <property name="text">
         <string>Last error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearLog">
        <property name="text">
         <string>Clear log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
     <item>
      <widget class="QTextBrowser" name="log"/>
     </item>
     <item>
      <widget class="QLabel" name="image">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
